harness/determined/profiler.py

```diff
@@ -364,6 +364,19 @@ def from_env(env: det.EnvContext, global_rank: int, local_rank: int) -> "Profile
             sync_timings=env.experiment_config.profiling_sync_timings(),
         )
 
+    @staticmethod
+    def _dummy_init():
```

Comment:
what is this for?  Why not create a DummyProfilerAgent instead?

Reply:
done